socorro/processor/signature_utilities.py

```diff
@@ -93,6 +93,63 @@ def __init__(self, config, quit_check_callback=None):
         self.fixup_comma = re.compile(r',(?! )')
 
     #--------------------------------------------------------------------------
+    def _is_exception(
+        self,
+        exception_list,
+        remaining_original_line,
+        line_up_to_current_position
+    ):
+        for an_exception in exception_list:
+            if remaining_original_line.startswith(an_exception):
+                return True
+            if line_up_to_current_position.endswith(an_exception):
+                return True
+        return False
+
+    #--------------------------------------------------------------------------
+    def _collapse(
+        self,
+        function,
```

Comment:
This put me off. Once I realized that it has nothing to do with callable functions, it made more sense. It should maybe be called "string".